recipes/libcurl/all/test_package/conanfile.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
@uilianries the same for this. Why it was not detected?

Reply:
The PR is green, don't add more hooks so it fails 😄

Reply:
@danimtb The [hook](https://github.com/conan-io/hooks/blob/master/hooks/conan-center.py#L223) doesn't check the test package ... we need to update it :smiling_imp: